docs/usage/configuration-options.md

```diff
@@ -119,6 +119,12 @@ See [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-s
 
 If configured, Renovate will take a random sample of given size from assignees and assign them only, instead of assigning the entire list of `assignees` you have configured.
 
+## autoReplaceGlobalMatch
+
+Setting this to `false` will replace only the first match during replacements updates.
+
+Disabling this is useful for situations where values are repeated within the dependency string, such as when the `currentVersion` is also featured somewhere within the `currentDigest`, but you only want to replace the first instance.
+
```

Comment:
Would you mind adding an example for each one of the options?

Reply:
@lmilbaum - not at all.  I've pushed an update with a draft scenario.  Appreciate some feedback on it and how it can be made less complex